napari/components/add_layers_mixin.py

```diff
@@ -155,91 +159,65 @@ def add_image(
         elif data is None:
             data = io.magic_imread(path)
 
+        # doing this here for IDE/console autocompletion in add_image function.
```

Comment:
Just curious, how does this relate to IDE autocomplete? It looks to me like this is useful in the for loop below

Reply:
that's just there for a newcomer that might look at that and say "why didn't you just put `**kwargs` in the function signature and remove all this extra code?  maybe it's either unnecessary or could be clearer

Reply:
oooh, i get it, lol. that's fine